app/globals.css

```diff
@@ -48,8 +47,6 @@ ol {
 /* Global Styles */
 body {
   line-height: 1.6;
-  background-color: #f8f9fa;
```

Comment:
p2: why change style?

if you want better style, you should give anotion pr

Reply:
In the case of Windows, I noticed that the transparent property for `::-webkit-scrollbar-trac` is not applied(transparent) due to the background color.

I will work on it with a screenshot in the next PR.